src/utils/apiUtils.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+
+const apiHome = axios.create({
+    baseURL:"https://elearningnew.cybersoft.edu.vn/api/"
+})
+
+apiHome.interceptors.request.use((config) => {
+    config.headers = {

```

Comment:
the output should be difference from input, please read this: https://eslint.org/docs/rules/no-param-reassign